Task 8/script.js

```diff
@@ -5,3 +5,24 @@ subtraction(a, b) - priima du skaičius ir grąžina jų skirtumą.
 multiplication(a, b) - priima du skaičius ir grąžina jų daugybos rezultatą;
 division(a, b) - priima du skaičius ir grąžina jų dalybos rezultatą;
 ------------------------------------------------------------------------------------ */
+
+function Calculator(num1, num2, op){
+  this.num1 = num1;
```

Comment:
function constructor should create 4 methods, not just one with switch. Also no need to accept any arguments, every method should accept num1, num2 themselves